src/components/WalletDropdown/DefaultMenu.tsx

```diff
@@ -27,7 +27,7 @@ const ConnectButton = styled(ButtonPrimary)`
 `
 
 const Divider = styled.div`
-  border: 1px solid ${({ theme }) => theme.backgroundOutline};
+  border-bottom: 1px solid ${({ theme }) => theme.backgroundOutline};
```

Comment:
This is a duplicated component, with Transactionmenu.tsx (which already correctly uses border-bottom). Can you make a new `HorizontalRule.tsx` component (and have it export an aliased `Divider` element to make it more searchable) and use that?

Reply:
Fine to do this as a follow-up, too.